homeassistant/components/home_connect/entity.py

```diff
@@ -83,33 +81,24 @@ def available(self) -> bool:
         as event updates should take precedence over the coordinator
         refresh.
         """
-        return self._attr_available
+        return self.appliance.info.connected
```

Comment:
We're setting `self._attr_available` here:
https://github.com/home-assistant/core/blob/70012badea4e765b9197f7e3c7212293db155472/homeassistant/components/home_connect/switch.py#L196

I think we need to still also take this attribute into consideration.